app/decorators/case/sar/offender_decorator.rb

```diff
@@ -23,7 +23,11 @@ def request_methods_for_display
 
   def rejected_reasons_descriptions
     rejected_reasons.map { |reason|
-      Case::SAR::Offender::REJECTED_REASONS[reason]
-    }.append(other_rejected_reason).compact.join("<br>")
+      if reason != "other"
+        Case::SAR::Offender::REJECTED_REASONS[reason]
+      else
+        "Other: #{other_rejected_reason}"
+      end
```

Comment:
Just thinking about this a little more and my personal preference would be to try to use a positive IF statement where possible. Also it does makes sense to still use the value from the REJECTED_REASONS hash as you were doing originally so maybe something like:
```suggestion
      suffix = ": #{other_rejected_reason}" if reason == "other"
      "#{Case::SAR::Offender::REJECTED_REASONS[reason]}#{suffix}"
```